yarn.lock

```diff
@@ -3506,10 +3436,10 @@ got@^6.7.1:
     url-parse-lax "^1.0.0"
 
 graceful-fs@^3.0.0:
-  version "3.0.11"
-  resolved "https://registry.yarnpkg.com/graceful-fs/-/graceful-fs-3.0.11.tgz#7613c778a1afea62f25c630a086d7f3acbbdd818"
+  version "3.0.12"
+  resolved "https://registry.yarnpkg.com/graceful-fs/-/graceful-fs-3.0.12.tgz#0034947ce9ed695ec8ab0b854bc919e82b1ffaef"
   dependencies:
-    natives "^1.1.0"
+    natives "^1.1.3"
```

Comment:
I remember we fixed the version of natives to a lower value (1.1.1 -> 1.1.0), but I do not remember why. I needed to update it to run gulp tasks on node 10.